test/e2e/multi_nic_test.go

```diff
@@ -36,7 +36,7 @@ const (
 	additionalSubnetVarKey = "NUTANIX_ADDITIONAL_SUBNET_NAME"
 )
 
-var _ = Describe("Nutanix Subnets [PR-Blocking]", func() {
+var _ = Describe("Nutanix Subnets", func() {
```

Comment:
same

Reply:
this is debatable but if our quick start test passes that means this is working in a way. hence not pr blocker